### PR TITLE
Create Shrey-Mehra.md

### PR DESCRIPTION
**Pull request by Shrey-Mehra**
## Describe your project in 1-3 sentences

I, along with @SHREEYA2007, am building the project which is aimed at saving lives caused due to delays in treatment.

## Hack Club Slack handle

@ Shrey Mehra
@ Shreeya Rani

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by SHREEYA2007** [comment]
Hi, I am Shreeya!

**Comment by bellesea** [comment]
> Hi, I am Shreeya!

hey @SHREEYA2007 can you please confirm that you're

[ ] I am a current high school student (or younger).
[ ] I have submitted an RSVP at https://hackclub.com/winter
[ ] I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
[ ] I have self-reviewed my project proposal.

File: Shrey-Mehra.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+name: "@Shrey-Mehra"
```
Comment (bellesea) [inline Shrey-Mehra.md:2]:
```suggestion
name: ["@Shrey-Mehra", "@SHREEYA2007"]
```

**Review by bellesea — CHANGES_REQUESTED** [review]
hiiii @Shrey-Mehra and @SHREEYA2007 :) this is REALLLYYY COOOL!

i have a few questions:
- what "medical properties" are you referring to?
- could you please elaborate on what the user interface would look like and how you plan on linking your device to the interface?
- what are you 3d printing within this project? 
- could you please elaborate on the roles of the items within your budget (eg. 3d printer, pi, sensors, etc.) an how they'd help make your machine work?

please respond to these questions by committing additional details to your file :) ideally we'd be able to visualise you building this project!

thank you :)

**Comment by SHREEYA2007** [comment]
Hi @bellesea , is there any particular procedure to confirm that 
[ ] I am a current high school student (or younger).
[ ] I have submitted an RSVP at https://hackclub.com/winter
[ ] I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
[ ] I have self-reviewed my project proposal?

**Comment by bellesea** [comment]
> Hi @bellesea , is there any particular procedure to confirm that [ ] I am a current high school student (or younger). [ ] I have submitted an RSVP at https://hackclub.com/winter [ ] I am a member of the Hack Club Slack (Join at https://hackclub.com/slack) [ ] I have self-reviewed my project proposal?

please just respond with the list above and adding an [x] within it to check it off :)

**Comment by SHREEYA2007** [comment]
> > Hi, I am Shreeya!
> 
> hey @SHREEYA2007 can you please confirm that you're
> 
> [x] I am a current high school student (or younger). [x] I have submitted an RSVP at https://hackclub.com/winter [x] I am a member of the Hack Club Slack (Join at https://hackclub.com/slack) [x] I have self-reviewed my project proposal.

**Review by Shrey-Mehra — COMMENTED** [review]
We have Done with all the changes.

please let us know if any changes required 

Regards
Shrey Mehra

**Comment by bellesea** [comment]
hey @Shrey-Mehra i saw that you added a pi screen, could you please add under your plan how you will be using it? thank you sm :)

**Comment by Shrey-Mehra** [comment]
Hey @bellesea actually i need display for displaying the data or we can say as a interface
Thank you

**Comment by bellesea** [comment]
> Hey @bellesea actually i need display for displaying the data or we can say as a interface Thank you

great! please commit this detail to your plan :)

**Comment by Shrey-Mehra** [comment]
Sure I'll update it asap
Thank you sm:)

**Comment by Shrey-Mehra** [comment]
Hey @bellesea i update about screen in plan
Thank you:)

**Review by hackliv — APPROVED** [review]
This sounds so awesome! I'd love to hear more about it. 😃 I'm dming you both now on Slack to chat about next steps for the grant!

**Pull request merged**